status/types/fleet.nim

```diff
@@ -9,13 +9,16 @@ type
     Test = "eth.test",
     WakuV2Prod = "wakuv2.prod"
     WakuV2Test = "wakuv2.test"
-
+    GoWakuTest = "go-waku.test"
```

Comment:
Wonder if this naming is good. Does this imply the other WakuV2 nodes are not "go" nodes? Is it even relevant here?

Reply:
The other WakuV2 nodes are not go-waku nodes ;)   

The naming was done matching https://fleets.status.im/ . `go-waku.fleet` is a fleet of `go-waku` only nodes that have relay + rendezvous protocol enabled, and it was created for enabling the discovery of new peers with rendezvous protocol. Ideally we would not have this distinction between fleet types and just check for the capabilities of the peers in the waku fleet, but this is a temporary solution to have a discovery mechanism available while discovery V5 is being worked on nim-waku